src/libopensc/pkcs15-openpgp.c

```diff
@@ -216,7 +216,7 @@ sc_pkcs15emu_openpgp_init(sc_pkcs15_card_t *p15card)
 
 		pin_info.auth_type = SC_PKCS15_PIN_AUTH_TYPE_PIN;
 		pin_info.auth_id.len      = 1;
-		pin_info.auth_id.value[0] = i + 1;
+		pin_info.auth_id.value[0] = pin_cfg[i].reference;
```

Comment:
Hi Nguyễn,

> Could you please explain me more about this change?

Did you read commit message for this change?
It's commit 0a6c1c4fb3fee9b27cd71fd97e7e8b40a880286d .